### DISCUSSION TITLE
Support for page status and labels in Confluence

### DISCUSSION THREAD

**Discussion by bneradil** [quarto-dev/quarto-cli#10912] [Q&A] [answered]
### Description

Is there current or planned support for [page status](https://support.atlassian.com/confluence-cloud/docs/add-a-status-to-your-page-or-blog/) when publishing into confluence?

And any plans for implementing page labels ([Issue 4478](https://github.com/quarto-dev/quarto-cli/issues/4478))?

**Comment by cscheid** [comment, marked as answer]
No plans - we don't currently have the resources to invest into the format (but pull requests are always welcome.)
